### PR TITLE
Form structure for Log in view

### PR DESCRIPTION
**Pull request by cortezroberto**
# Description
 -  We added the structure for the Log in form.
 - Fields and components creation.
 
 # Notes
 - The handleSignIn function is just for guide purposes, we are going to create the handleLogIn function in another issue.

### CODE REVIEW

File: FlightReservation/src/screens/LogIn.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                />
+                <Text>or</Text>
+                <Button title="Log in with Google" />
+                <Text>Don't have an account?</Text>
+                <Text>Sign up</Text>
+              </View>
+            </View>
+          )}
+        </Formik>
+      </Provider>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:68]: The use of the provider with the store should be global and used once please move the provider as a wrapper for the app in root/App.js provider also is present in signup.js (it should exist only an instance in the project) please read the redux documentation or ask for advice

**Review by DemisRincon — CHANGES_REQUESTED** [review]
Please review the request of changes

File: FlightReservation/src/screens/LogIn.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+//   const [emailInUseError, setEmailInUseError] = useState(false);
+
+  const handleSignIn = values => {
+    const {email, password} = values;
+    setLoading(true);
+    signInWithNameEmailAndPassword(email, password)
+      .then(() => setEmailInUseError(false))
+      .catch(() => setEmailInUseError(true))
+      .finally(() => setLoading(false));
+  };
```
Comment (YilKanda) [inline FlightReservation/src/screens/LogIn.js:23]: If you can try to pass this function to a separate file, for example Validations.js, so as not to overload this component.

File: FlightReservation/src/screens/LogIn.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import {store} from '../store/store';
+import {Formik} from 'formik';
+import TextField from '../components/form/TextField';
+import PasswordField from '../components/form/PasswordField';
+import {signUpValidationSchema} from '../schemas/signUpSchema';
+import {signInWithNameEmailAndPassword} from '../helpers/firebaseSignUp';
+
+const LogIn = function () {
+  const [loading, setLoading] = useState(false);
+//   const [emailInUseError, setEmailInUseError] = useState(false);
```
Comment (YilKanda) [inline FlightReservation/src/screens/LogIn.js:14]: If you don't occupy this line, delete it.

File: FlightReservation/src/screens/LogIn.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      .then(() => setEmailInUseError(false))
+      .catch(() => setEmailInUseError(true))
+      .finally(() => setLoading(false));
+  };
+
+  return (
+    <SafeAreaView>
+      <Provider store={store}>
+        {loading && <Text>Loading...</Text>}
+        {/* {emailInUseError && <Text>Email in use. Use a diferent email</Text>} */}
```
Comment (YilKanda) [inline FlightReservation/src/screens/LogIn.js:29]: If you don't occupy this line, delete it.

**Review by YilKanda — COMMENTED** [review]
Try to use internalization instead of static texts.

You can make use of libraries like this:

[react-native-i18n](https://www.npmjs.com/package/react-native-i18n)

**Review by hancel-avila — APPROVED** [review]
Nice work!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by DemisRincon — APPROVED** [review]

**Pull request merged**